### PR TITLE
add tendrl_unit_test_setup.yml

### PR DESCRIPTION
**Pull request by fbalak**
Add playbook that sets machine up for unit testing.

### CODE REVIEW

File: tendrl_unit_test_setup.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      retries: 5
+      delay: 5
+    - name: WORKAROUND configure selinux to permissive mode
+      selinux:
+        state: permissive
+        policy: targeted
+  roles:
+    - { role: epel, epel_enabled: 1 }
+    - qe-tendrl-repo
+  post_tasks:
```
Comment (mbukatov) [inline tendrl_unit_test_setup.yml]: You should also make sure `pytest` is installed.

File: tendrl_unit_test_setup.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# * Install Tendrl server.
+# * Install Tendrl storage nodes.
+# * Install Gluster nodes.
+# * Configure gluster trusted storage pool.
+# * Install Ceph nodes.
+# * Preconfigure ceph-ansible.
+- hosts: usm_test
+  remote_user: root
+  vars:
+    etcd_ip_address: "{{ ansible_default_ipv4.address }}"
```
Comment (mbukatov) [inline tendrl_unit_test_setup.yml]: This variable doesn't need to be there.

File: tendrl_unit_test_setup.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+# Perform following tasks, when applicable
+# (e.g. there are nodes in gluster or ceph_* group)
+# * Configure firewall
+# * Install Tendrl server.
+# * Install Tendrl storage nodes.
+# * Install Gluster nodes.
+# * Configure gluster trusted storage pool.
+# * Install Ceph nodes.
+# * Preconfigure ceph-ansible.
```
Comment (mbukatov) [inline tendrl_unit_test_setup.yml]: This comment seems out of touch with this playbook file :)

File: tendrl_unit_test_setup.yml [hunk 4]
```diff
@@ -29,6 +16,9 @@
     - { role: epel, epel_enabled: 1 }
     - qe-tendrl-repo
   post_tasks:
+    - name: Install pytest
+      pip:
+        name: 'pytest'
```
Comment (mbukatov) [inline tendrl_unit_test_setup.yml]: Could you install pytest from rpm package instead?

File: tendrl_unit_test_setup.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+# Sets up machine so there can be performed Tendrl unit tests
+# Installs all tendrl packages and pytest
+- hosts: usm_test
```
Comment (dahorak) [inline tendrl_unit_test_setup.yml]: I'm thinking about better/more descriptive name for the `usm_test` group...  what about something like `unittest_node`?

Comment (mbukatov) [inline tendrl_unit_test_setup.yml]: That is a good point.

Comment (fbalak) [inline tendrl_unit_test_setup.yml]: agreed

**Pull request merged**